README.md

```diff
@@ -52,7 +52,7 @@ You can install the Doctrine Coding Standard as a composer dependency to your pa
 Just add the following block to your project's `composer.json` file:
 
 ```bash
-$ php composer require doctrine/coding-standard:^3.0
+$ php composer require doctrine/coding-standard:^4.0
```

Comment:
Since we now follow SemVer in this package, the constraint can be completely removed here.